jerry-core/api/jerry.c

```diff
@@ -671,14 +663,19 @@ jerry_run (const jerry_value_t func_val) /**< function to run */
     return jerry_throw (ecma_raise_type_error (ECMA_ERR_MSG (wrong_args_msg_p)));
   }
 
-  ecma_object_t *obj_p = ecma_get_object_from_value (func_val);
+  ecma_object_t *object_p = ecma_get_object_from_value (func_val);
```

Comment:
I'd also update the doc of this function here and in the API doc a bit, since we no longer title script/modules functions anymore.

`Run an EcmaScript function created by jerry_parse` -> `Run a script/module created by jerry_parse`
`/**< function to run */` -> /**< script/module to run */